src/aat/java/uk/gov/hmcts/ccd/userprofile/befta/UserProfileTestAutomationAdapter.java

```diff
@@ -1,109 +1,13 @@
 package uk.gov.hmcts.ccd.userprofile.befta;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import java.io.File;
-import java.util.HashMap;
-import java.util.Map;
-
 import io.restassured.RestAssured;
-import io.restassured.builder.RequestSpecBuilder;
-import io.restassured.response.Response;
-import io.restassured.specification.RequestSpecification;
-import uk.gov.hmcts.befta.*;
-import uk.gov.hmcts.befta.data.UserData;
-import uk.gov.hmcts.befta.exception.FunctionalTestException;
+import uk.gov.hmcts.befta.DefaultTestAutomationAdapter;
 
 public class UserProfileTestAutomationAdapter extends DefaultTestAutomationAdapter {
```

Comment:
@MSancaktutar and I discussed removing of user profile func tests importing dependency as it won't work on all environments (preview vs importing on AAT). We should be putting the user profiles using the same API and then trying to get the profiles back as the actual functional test verification.

Reply:
Yes, we should actually push required data to the current application under test.